### PR TITLE
lp earning: enable all chains

### PR DESCRIPTION
**Pull request by viet-nv**

### CODE REVIEW

File: .env.adpr
Comment (namgold) [inline .env.adpr:1]: revert

Comment (viet-nv) [reply .env.adpr:1]: approve ddi banj

File: src/pages/MyEarnings/MyEarningStats/index.tsx [hunk 1]
```diff
@@ -79,7 +80,10 @@ const MyEarningStats = () => {
   const tokensByChainId = useAppSelector(state => state.lists.mapWhitelistTokens)
 
   const elasticEarningQueryResponse = useGetElasticEarningQuery({ account, chainIds: selectedChainIds })
-  const elasticLegacyEarningQueryResponse = useGetElasticLegacyEarningQuery({ account, chainIds: selectedChainIds })
+  const elasticLegacyEarningQueryResponse = useGetElasticLegacyEarningQuery({
+    account,
+    chainIds: selectedChainIds.filter(item => !COMING_SOON_NETWORKS_FOR_MY_EARNINGS_LEGACY.includes(item)),
+  })
```
Comment (namgold) [inline src/pages/MyEarnings/MyEarningStats/index.tsx:86]: Do we need to memo the param? Can RTK handle this?

**Review by namgold — APPROVED** [review]

File: src/pages/MyEarnings/MyEarningStats/index.tsx (hunk 1 above)
Comment (viet-nv) [reply src/pages/MyEarnings/MyEarningStats/index.tsx:86]: it can

**Pull request merged**